### PR TITLE
TT-12306, addressed changes suggested in the tyk-docs PR

### PR DESCRIPTION
**Pull request by andrei-tyk**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

___

### **PR Type**
enhancement, bug fix

___

### **Description**
- Added `// nolint` comments to alias declarations in `apidef/oas/event.go` and `apidef/oas/time.go` to prevent linting issues.
- Corrected typos in comments within `config/rate_limit.go` to improve clarity and accuracy.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>event.go</strong><dd><code>Add nolint comments to alias declarations in event.go</code>&nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

apidef/oas/event.go

- Added `// nolint` comments to alias declarations.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk/pull/6377/files#diff-528a9f5b311ff21c0b3a9b273e61398209ca8b51550327e4d437bba81e49d577">+2/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>time.go</strong><dd><code>Add nolint comment to alias declaration in time.go</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

apidef/oas/time.go

- Added `// nolint` comment to alias declaration.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk/pull/6377/files#diff-56f5ed8764cba1fab5cc4615f02a769b2c9124e69b5972cb1b7921d1bded577b">+1/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Bug fix</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>rate_limit.go</strong><dd><code>Fix typos in rate limiter configuration comments</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

config/rate_limit.go

- Corrected typos in comments for rate limiter configurations.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk/pull/6377/files#diff-375bf116f8d6527c50d7591d7cb01e8f821b22df4a4ca18b4da4c6f0d526f18e">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Review by dcs3spp — APPROVED** [review]
Thanks @andrei-tyk I have approved PR

**Comment by dcs3spp** [comment]
If ReadableDuration is and alias also can that also be hidden before the merging?

**Review by dcs3spp — COMMENTED** [review]
If ReadableDuration is an alias also can that also be hidden before merging?

**Review by dcs3spp — APPROVED** [review]
Thanks @andrei-tyk I've approved the update

File: apidef/oas/event.go [hunk 1]
```diff
@@ -10,9 +10,11 @@ import (
 )
 
 // Kind is an alias maintained to be used in imports.
+// nolint
```
Comment (titpetric) [inline apidef/oas/event.go:13]: This is probablly wrong,...

Comment (titpetric) [reply apidef/oas/event.go:13]: Remove `// nolint` hints, if the intent is to skip some symbols, we need to adjust the CI action.
We already skip a few symbols from getting exported to docs here:
https://github.com/TykTechnologies/exp/blob/main/.github/workflows/tyk-docs.yml#L154

**Review by titpetric — CHANGES_REQUESTED** [review]

**Review by lghiur — APPROVED** [review]

**Pull request merged**